components/inputs/input-checkbox.js

```diff
@@ -199,27 +196,33 @@ class InputCheckbox extends InputInlineHelpMixin(FocusMixin(SkeletonMixin(RtlMix
 		};
 		const ariaChecked = this.indeterminate ? 'mixed' : undefined;
 		const disabled = this.disabled || this.skeleton;
-		const offscreenContainer = this.description ? html`<div class="d2l-offscreen" id="${this._descriptionId}">${this.description}</div>` : null;
-		const ariaDescribedByIds = `${this.description ? this._descriptionId : ''} ${this._hasInlineHelp ? this._inlineHelpId : ''}`.trim();
+		const offscreenContainer = this.description ? html`<div class="d2l-offscreen" id="${this.#descriptionId}">${this.description}</div>` : null;
+		const ariaDescribedByIds = `${this.description ? this.#descriptionId : ''} ${this._hasInlineHelp ? this.#inlineHelpId : ''}`.trim();
+		const disabledTooltip = disabled && this.disabledTooltip ?
+			html`<d2l-tooltip align="start" class="vdiff-target" for="${this.#inputId}" ?force-show="${this._isHovered}" position="top">${this.disabledTooltip}</d2l-tooltip>` :
+			nothing;
 		return html`
-			<label>
+			<label @mouseleave="${this.#handleMouseLeave}" @mouseenter="${this.#handleMouseEnter}">
```

Comment:
Needed to track the user's hover state separately and force tooltip with force-show because we want to display it when they hover over the checkbox text, not just the input.

Reply:
Seems like there is an edge case where tooltips could collide (more than one showing at the same time) if the user focuses to show tooltip 1, and then uses then hovers with mouse to show tooltip 2, since I don't think tooltips rendered with `force-show` will be hidden when another tooltip is displayed. Maybe is an edge-case that won't come up / we can live with, but figured I would mention it.

My default position would be to avoid `force-show` due to collision potential, but the probability largely depends on what else the consumer has in their UI. [This PR](https://github.com/BrightspaceUI/core/pull/3310) has a couple screenshots.

Reply:
Yeah I agree it's not ideal, but unless we want to add functionality to tooltip that would separate the `for` target from the show/hide/position target I'm not sure what else to do.

Reply:
That seems like it would be the ultimate in flexibility, but I don't think it's worth the added tooltip complexity. At least not so far.

Reply:
Agreed, especially since focus would still need to be triggered from the `for` target (the checkbox input) but hover/position on something else. Super complex!